File: libJava/com/earndero/sqlwrapper/java/ReadableDatabase.java
package com.earndero.sqlwrapper.java;

import java.io.Closeable;
import java.io.IOException;

public class ReadableDatabase extends Database implements Closeable {
    public ReadableDatabase(Context context, String dbname) throws Exception {
        super(dbname);
    }

    @Override

    public void close() throws IOException {

    }
}

File: libJavaKt/com/earndero/sqlwrapper/javakt/ReadableDatabase.kt
package com.earndero.sqlwrapper.javakt

import kotlin.Throws
import java.io.IOException
import java.io.Closeable

class ReadableDatabase(context: Context?, dbname: String) : Database(dbname), Closeable {
    @Throws(IOException::class)
    override fun close() {
    }
}